platformio.ini

```diff
@@ -154,6 +154,22 @@ platform          = https://github.com/p3p/pio-nxplpc-arduino-lpc176x/archive/ma
 framework         = arduino
 board             = nxp_lpc1768
 build_flags       = -DTARGET_LPC1768 -DU8G_HAL_LINKS -IMarlin/src/HAL/HAL_LPC1768/include -IMarlin/src/HAL/HAL_LPC1768/u8g ${common.build_flags}
+  -funwind-tables
+  -mpoke-function-name
+lib_ldf_mode      = off
+extra_scripts     = Marlin/src/HAL/HAL_LPC1768/upload_extra_script.py
+src_filter        = ${common.default_src_filter} +<src/HAL/HAL_LPC1768>
+monitor_speed     = 250000
+lib_deps          = https://github.com/MarlinFirmware/U8glib-HAL/archive/dev.zip
+  TMC2130Stepper@>=2.2.1
+
+[env:LPC1768_debug_and_upload]
```

Comment:
Not sure why this has been readded

Reply:
See previous comment about the pio env stuff!